src/fromager/sources.py

```diff
@@ -73,7 +78,7 @@ def resolve_sdist(req, sdist_server_url, only_sdists=True):
 
 def default_download_source(ctx, req, sdist_server_url):
     "Download the requirement and return the name of the output path."
-    url, version = resolve_sdist(req, sdist_server_url)
+    url, version = resolve_dist(req, sdist_server_url, include_wheels=False)
```

Comment:
since we are refactoring concept of sdist --> dist we should probably also explicitly say `include_sdists=true`, no?

Reply:
sure, that's reasonable, I'll post another PR with that